documentation/src/main/asciidoc/stories/assembly_wildfly_modules.adoc

```diff
@@ -0,0 +1,11 @@
+[id='ispn_modules']
+:context: server
+= Using {brandname} in {wildflybrandname} applications
+
+{wildflybrandname} includes {brandname} modules that you can use in {wildflybrandname} applications.
```

Comment:
not sure whether we should mention the WF version which includes core only and from which version the full API is available (like query counter etc)
Or we simple rely on the assumption that it is the latest

Reply:
I think it'd be good to mention the versions. Can you please help me with the info?